Documentation/Signoffs/sensor_subsystem.md

```diff
@@ -0,0 +1,43 @@
+
+# Sensor Subsystem
+
+## **Function:**
+
+The goal of this subsystem is to accurately and quickly gather several data points to determine the
+distance, speed, and position of the target as it slides down the fishing line.
+
+## **Constraints:**
+
+| No. | Constraint                                                                                                                                     | Origin            |
+|-----|------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| 1   | The sensor shall be supplied 5 V via USB from the Jetson Nano processor                                                                        | System Constraint |
+| 2   | The sensor, when mounted, shall not exceed the 1' x 1' x 1' size constraint of the launcher                                                    | Conceptual Design |
+| 3   | The sensor shall be able to retrieve at least 2 data points within 1.95 s in order to calculate speed                                          | Rulebook          |
+| 4   | SOMETHING ABOUT RESOLUTION AND LINE DETECTION | Conceptual Design |
```

Comment:
Why is this here? If the design isn't ready to be reviewed, don't request a review.